### DISCUSSION TITLE
How do I refresh the chat component?

### DISCUSSION THREAD

**Discussion by ekandreas** [namumakwembo/wirechat#18] [Q&A]
I want to generate computed answers and send som kind of update to the receiver.
How do I solve that?

**Comment by namumakwembo** [comment]
If you're looking to handle computed answers and notify the receiver, I suggest referring to the **[https://wirechat.namuio.com/docs/events](https://wirechat.namuio.com/docs/events)**. It explains how to use events like `MessageCreated` and `NotifyParticipant` to update the chat in real time.
